### PR TITLE
Remove webpack and merge

### PR DESCRIPTION
**Pull request by ssteinerx**
There are a couple of small hunks I'd leave (`@goldhand` line in contributors file), and the Postgresql line in cookiecutter.json, but I think that, otherwise, this does the job.

I'd like someone more familiar with the release procedure etc. to check this over, run tests etc. or let me know where to look for such a procedure.

ssteinerX

### CODE REVIEW

**Comment by pydanny** [comment]
Aside from the failing tests, this looks pretty good.Hopefully I can help with that today. 😄 

Yes, @goldhand needs to always remain in the contributors' file.

**Comment by ssteinerx** [comment]
I didn't add myself either, now that I think about it.

File: cookiecutter.json [hunk 1]
```diff
@@ -18,8 +18,7 @@
     "use_docker": "y",
     "use_heroku": "n",
     "use_compressor": "n",
-    "postgresql_version": ["9.5", "9.4", "9.3", "9.2"],
```
Comment (jayfk) [inline cookiecutter.json]: this shouldn't be removed.

Comment (ssteinerx) [inline cookiecutter.json]: @jayfk Yah, I added a note to that effect in the PR.

**Comment by ssteinerx** [comment]
Fixed two errors as noted in pull request and as pointed out by @jayfk.

File: cookiecutter.json (hunk 1 above)
Comment (ssteinerx) [inline cookiecutter.json]: And then I fixed it.

**Comment by ssteinerx** [comment]
Trying to figure out test running, accidentally generated cookiecutter_django..egg-info, wasn't excluded by .gitignore, made it so.

**Comment by ssteinerx** [comment]
Can anyone more familiar with the test suite point me at the source of what appears to be an errant tag in one of the Jinja templates (from other errors, this one's particularly obtuse).

Full test failure output at: [travis](https://travis-ci.org/pydanny/cookiecutter-django/jobs/158292929).

This a local one-off:

`$ tox -e py -- -k test_default_configuration`

```
____________________________________________________ test_default_configuration _______

cookies = <pytest_cookies.Cookies object at 0x104d0d510>
context = {'author_name': 'Test Author', 'description': 'A short description of the project.', 'domain_name': 'example.com', 'email': 'test@example.com', ...}

    def test_default_configuration(cookies, context):
        result = cookies.bake(extra_context=context)
>       assert result.exit_code == 0
E       assert -1 == 0
E        +  where -1 = <Result TemplateSyntaxError("Encountered unknown tag 'endif'.",)>.exit_code

tests/test_cookiecutter_generation.py:53: AssertionError
```

**Comment by jayfk** [comment]
> Can anyone more familiar with the test suite point me at the source of what appears to be an errant tag in one of the Jinja templates (gleaned from other errors, this one's particularly obtuse).

That comes from `{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/base.html`. The template is pretty insane. It switches between the cookiecutter context and the django context back and forth. Keep fiddling :)

**Comment by ameistad** [comment]
I think there is an lonely {% endif %} on line 86 in cookiecutter-django/{{cookiecutter.project_slug}}/README.rst that is causing the error.

**Comment by ssteinerx** [comment]
I'm sorry, @ameistad, @jayfk there were no winners in our *find the lonely endif* contest.  

It was Line 58 in README.rst and, fortunately, it was pretty unambiguous with only one `% if %` above line 5 (with its corresponding `% endif %` blessedly nearby).

**Comment by pydanny** [comment]
This is part of the resolution for #774

**Pull request merged**